CMakeLists.txt

```diff
@@ -366,10 +370,10 @@ if(GCC OR CLANG)
     else()
       set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wextra")
       set(C_CXX_FLAGS "${C_CXX_FLAGS} -Wall -fvisibility=hidden -fno-common")
+      set(CMAKE_ASM_FLAGS "${CMAKE_ASM_FLAGS} -Wno-newline-eof")
```

Comment:
not a concern, but why don't we want this for MSVC?

Reply:
It's apparently not a valid parameter on MSVC -- I was getting warning due to it during the build.